build/Android/XplatCppSdk/src/main/java/com/microsoft/xplatcppsdk/HttpRequest.java

```diff
@@ -30,47 +31,79 @@
     }
 
     public HttpRequest() {
-        requestBuilder = new Request.Builder();
-        requestCompleted = false;
+        this.requestBuilder = new Request.Builder();
     }
 
-    public static HttpRequest createRequest() {
-        return new HttpRequest();
+    public void setMethod(String method) {

```

Comment:
We only using POST, why we need it?

Reply:
This is bridge file and it is usually transparently working. And the method can be change anytime if the PlayFab api is changed.So this should be decided in PlayFab side not bridge.